webpack.config.js

```diff
@@ -180,5 +181,26 @@ const packageReportConfig = {
     target: 'node',
 };
 
+const packageUIConfig = {
+    entry: {
+        ui: [path.resolve(__dirname, 'src/packages/accessibility-insights-ui/index.ts')],
+    },
+    module: { rules: [tsRule, scssRule(false)] },
+    externals: [nodeExternals()],
+    plugins: commonPlugins,
+    resolve: commonConfig.resolve,
+    name: 'package-ui',
+    mode: 'development',
+    devtool: false,
+    output: {
+        path: path.join(__dirname, 'package/ui/bundle'),
+        filename: '[name].bundle.js',
+        pathinfo: false,
+        library: '[name]',
+        libraryTarget: 'umd',
+    },
+    target: 'node',
+};
+
 // For just one config, use "webpack --config-name dev", "webpack --config-name prod", etc
-module.exports = [devConfig, prodConfig, unifiedConfig, packageReportConfig];
+module.exports = [devConfig, prodConfig, unifiedConfig, packageReportConfig, packageUIConfig];
```

Comment:
Comment for a different PR: similar to the grunt file... our webpack config keeps growing over time... I guess it's time we ask ourselves: should we split this file into smaller ones?

Reply:
I agree but not sure I want to take this on here.